REQUIRE

```diff
@@ -1,8 +1,8 @@
-julia 0.6
-Compat 0.33.0
-Distributions 0.10.0
+julia 0.7-beta2
+Compat 0.60.0
```

Comment:
I don't think this is using `Compat` anymore.

Reply:
Okay.  That came from one of @dmbates's commits, so I wasn't going to get rid of it.  But I think you're right.